docs/03-tutorials/00-api-exposure/apix-02-setup-custom-domain-for-workload.md

```diff
@@ -1,71 +1,63 @@
 ---
-title: Use a custom domain to expose a workload
+title: Set up a custom domain for a workload
 ---
 
 This tutorial shows how to set up your custom domain and prepare a certificate for exposing a workload. It uses Gardener [External DNS Management](https://github.com/gardener/external-dns-management) and [Certificate Management](https://github.com/gardener/cert-management) components.
+  >**NOTE:** Feel free to skip this tutorial if you use a Kyma domain instead of your custom domain.
 
 ## Prerequisites
 
-To follow this tutorial, use Kyma 2.0 or higher.
-
 If you use a cluster not managed by Gardener, install the [External DNS Management](https://github.com/gardener/external-dns-management) and [Certificate Management](https://github.com/gardener/cert-management) components manually in a dedicated Namespace.
 
+This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create those workloads, follow the [Create a workload](./apix-01-create-workload.md) tutorial.
+
```

Comment:
what do you mean with sample function? the 01-create-workflow guide deploys just httpbin as described in:
https://raw.githubusercontent.com/istio/istio/master/samples/httpbin/httpbin.yaml

Reply:
my bad, didn't see the Function section.